core/iwasm/compilation/aot_llvm_extra.cpp

```diff
@@ -478,3 +479,4 @@ aot_apply_llvm_new_pass_manager(AOTCompContext *comp_ctx)
     }
 #endif
 }
+#endif
```

Comment:
Had better add a comment:
```suggestion
#endif /* end of LLVM_VERSION_MAJOR >= 12 */
```

Reply:
Done